OpenUpSA/wazimap-ng-ui
# Enable translation for search box placeholder

## Description
Enable translation for placeholders.
For translation to work on placeholder we have to add class `i18n-placeholder`

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-541

## How is it tested automatically?
There is an automated gui test in translations that check if placeholder for search box is translated

## How should a reviewer test it locally
Add translations for `Search for a location...` in profile configuration
It should be visible in FE when map is loaded

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
